src/components/calcite-input-time-picker/calcite-input-time-picker.tsx

```diff
@@ -0,0 +1,330 @@
+import {
+  Component,
+  Element,
+  Host,
+  VNode,
+  h,
+  Prop,
+  Listen,
+  Event,
+  EventEmitter,
+  State,
+  Method
+} from "@stencil/core";
+import { guid } from "../../utils/guid";
+import { formatNumberAsTimeString, Time } from "../calcite-time-picker/utils";
+import { Theme } from "../interfaces";
+
+@Component({
+  tag: "calcite-input-time-picker",
+  styleUrl: "calcite-input-time-picker.scss",
+  scoped: true
```

Comment:
why not shadow here?

Reply:
Form accessibility

Reply:
The shadowed `time-picker` component, where all the custom styles occur is where the encapsulation occurs.  The `input-time-picker` component is really only there to provide the form integration and expose the encapsulated time-picker for people to use.